opentype-shaping-devanagari.md

```diff
@@ -1192,6 +1214,22 @@ conjunct ligatures. These sequences must match "_Consonant_,Halant,_Consonant_".
 A sequence matching "_Consonant_,Halant,ZWJ,_Consonant_" or
 "_Consonant_,Halant,ZWNJ,_Consonant_" must not be flagged to form a conjunct.
 
+> Note: The presence of the "ZWJ" at the end of the sequence means
+> that the sequence will match the regular-expression test in stage 1
+> as the end of a syllable.
+> 
+> Because OpenType shaping features in `<dev2`> are defined as
+> applying only within an individual syllable, this means that the
+> presence of the "ZWJ" will automatically prevent the application of
+> a `cjct` feature by triggering the identification of a syllable
+> break between the two consonants.
```

Comment:
I could be misunderstanding the wording here, but (if I recall correctly) the presence of a "ZWJ" in a "Consonant, Halant, ZWJ, Consonant" sequence wouldn't trigger a break between the two consonants. A "ZWNJ" yes, but not a "ZWJ".

E.g. in the consonant syllable regex, we have `... (CN HALANT_GROUP)* CN ...`. The `HALANT GROUP` consumes the `Halant, ZWJ` (as `HALANT_GROUP = Z? _halant_ (_zwj_ _nukta_?)?`), thus making the sequence a single syllable.

Rather, I _think_ the presence of the "ZWJ" here inhibits the application of the `cjct` feature simply by being there.

Reply:
Right; that's just poor choice of phrasing on my part.

Reply:
I guess "regular-expression break" or something of that ilk would be more like the intent, but there's not really a pre-established term along those lines. Maybe this is just entirely unnecessary to comment on.